pkg/ami/api.go

```diff
@@ -57,6 +57,24 @@ func IsAvailable(api ec2iface.EC2API, id string) (bool, error) {
 	return *output.Images[0].State == "available", nil
 }
 
+// LookupRootDeviceName gets the root block device mapping
+func LookupRootDeviceName(api ec2iface.EC2API, id string) (string, error) {
+	input := &ec2.DescribeImagesInput{
+		ImageIds: []*string{&id},
+	}
+
+	output, err := api.DescribeImages(input)
+	if err != nil {
+		return "", errors.Wrapf(err, "unable to determine root device name for %q", id)
+	}
+
+	if len(output.Images) < 1 {
+		return "", errors.Wrapf(err, "unable to find %q", id)
```

Comment:
Perhaps make the log message more explicit: "unable to find root device %q"

Reply:
We have not even checked the root device yet, this error will happen if no AMI data is returned at all. If I change it to be root device specific would it not be a bit misleading?

Reply:
Ah, right, so this is the id of the image? then how about `unable to find image %q`?

Reply:
This has been merged into the IsAvailable function.